instrumentation/spring/spring-webflux-5.0/auto/src/test/resources/logback.xml

```diff
@@ -13,6 +13,6 @@
     <appender-ref ref="console"/>
   </root>
 
-  <logger name="io.opentelemetry.auto" level="debug"/>
+  <logger name="io.opentelemetry" level="debug"/>
```

Comment:
Do we need these individual configuration files then?

Reply:
ya, they are needed because these tests run Spring Boot, which overrides the logging level we set in AgentTestRunner. I added a comment to clarify.